carbon/assetlist.json

```diff
@@ -125,6 +173,18 @@
         "name": "Stride Staked OSMO",
         "display": "stOSMO",
         "symbol": "stOSMO",
+        "traces": [{
```

Comment:
this trace looks incorrect... should be chain_name: stride, base_denom: stuosmo. Not sure the channel ids between carbon and stride...